tendrl/ceph_integration/objects/rbd/atoms/create/__init__.py

```diff
@@ -13,6 +17,54 @@ def __init__(self, *args, **kwargs):
         super(Create, self).__init__(*args, **kwargs)
 
     def run(self):
+        # Checking if a new  pool has to be created or not rbd creation.
+        if self.parameters['Rbd.create_pool'] is True and \
```

Comment:
Is this verified? I dont see the "create_pool" attribute defined as a mandatory input for this Create rbd flow in definitions

If it is not mandatory, then you need to handle the absence of the same from self.parameters